### PR TITLE
back-to-top Fix

### PR DESCRIPTION
**Pull request by AmanBisht01**
[ Done ] The to top button's position should be dynamic w.r.t location of the page
[ Done ] On click, it should take us to the topmost section of each page
[ Done ] It should not appear on mobile screens
[ Done ] It should not appear on the top of the page (or in fact midway)

issue #857

### CODE REVIEW

File: src/components/Footer/Footer.css [hunk 1]
```diff
@@ -1,69 +1,69 @@
...
+@import "../../pages/Home/Home.css";
 
 /*Footer Base*/
 .footer {
-    padding-top: 10px;
-    font-family: "Poppins", sans-serif;
-    background-color: #121212;
+  padding-top: 10px;
+  font-family: "Poppins", sans-serif;
+  background-color: #121212;
```
Comment (smaranjitghose) [inline src/components/Footer/Footer.css:7]: Use variables for the colors. Name them appropriately. Declare separately for dark mode and light mode. Check the style.css in the master branch please

File: src/components/Footer/Footer.css [hunk 2]
```diff
@@ -105,347 +105,349 @@ form {
...
-    -moz-box-sizing: border-box;
-    width: 100%;
-    margin-top: 2rem;
-    padding: 3%;
-    background: #6a00bb;
-    border-bottom: 2px solid #6a00bb;
-    border-radius: 2px;
-    border-top-style: none;
-    border-right-style: none;
-    border-left-style: none;
```
Comment (smaranjitghose) [inline src/components/Footer/Footer.css:127]: Undo all the indentation/linting changes please

File: src/components/Footer/Footer.js [hunk 3]
```diff
@@ -1,74 +1,742 @@
...
+
+  const handleScroll = () => {
+    if (window.pageYOffset > showBelow) {
+      if (!show) setShow(true);
+    } else {
+      if (show) setShow(false);
+    }
+  };
+
+  useEffect(() => {
```
Comment (smaranjitghose) [inline src/components/Footer/Footer.js:23]: Add comments please. Poorly documented code is of no use in the long term

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

File: src/components/Footer/Footer.js [hunk 4]
```diff
@@ -1,74 +1,742 @@
-import React from 'react';
-import './Footer.css'
-import { FaQuoteLeft } from 'react-icons/fa';
-import { FaAngleUp } from 'react-icons/fa';
-const scrollToTop = () => {
-    window.scrollTo({
```
Comment (smaranjitghose) [inline src/components/Footer/Footer.js:6]: Undo all these linting/indentation changes

File: src/components/Footer/Footer.js [hunk 5]
```diff
@@ -1,74 +1,742 @@
...
+            <a
+              className="social-button twitter"
+              href="https://twitter.com/"
+              target="_blank"
+              rel="noreferrer"
+            >
+              <svg
+                className="twitter-icon-footer"
+                version="1.1"
+                id="Layer_1"
```
Comment (smaranjitghose) [inline src/components/Footer/Footer.js:727]: Undo these linitng/indentation changes

**Comment by AnirudhPanda** [comment]
Are you resolving the merge conflicts on this @AmanBisht01 ?

**Comment by AmanBisht01** [comment]
@AnirudhPanda Resolved conflicts.

**Pull request merged**